ui/src/components/utils/AutoComplete.vue

```diff
@@ -67,21 +73,8 @@
             </slot>
           </button>
         </li>
-
```

Comment:
Removed 'Begin typing to search' because the AutoComplete's behavior has always been to show some results when the AutoComplete is opened (before the user has typed anything).